packages/terra-filter-view/src/_FilterOption.module.scss

```diff
@@ -0,0 +1,84 @@
+@import '~terra-mixins';
```

Comment:
These variables will need to be scoped under terra-filter

Reply:
done here: https://github.com/cerner/terra-core/pull/1685/commits/53e42b9203ab32c413f61a567888ae79b090cb64#diff-ba1c49cc02020d82360fe5d0207cd3e4